src/new-components/text-link/text-link.module.css

```diff
@@ -0,0 +1,21 @@
+.container {
+    font-size: inherit;
+    font-weight: inherit;
+    font-family: inherit;
+    text-decoration: none;
+}
+
+.container:hover {
+    text-decoration: underline;
+}
+
+.container svg {
```

Comment:
I think the component can be more explicit in accepting an icon. What do you think about accepting an `icon` prop that expects a component with a className prop? This way, the component also guarantees the order in which the icon is displayed (ie. always on the left).

Reply:
I am precisely not wanting to assume an icon in a text link will be always on the left.